examples/spark/images/base/core-site.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+
+<configuration>
+  <property>
+    <name>fs.gs.impl</name>
+    <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem</value>
```

Comment:
This is specializing the backing storage to use GFS.  IMHO if you're creating an image for the community this should be abstracted.